.gitignore

```diff
@@ -11,5 +11,5 @@ release-tool
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+# Project-local glide cache, RE: https://github.com/github/gitignore/pull/2253
 .glide/
```

Comment:
Why do we need this? We don’t use glide.

Reply:
no reason that I can discern... https://github.com/github/gitignore/pull/2505/files shows it coming out ... presume we picked it up by default.. will remove.